### PR TITLE
Make swift compile instead of just running

### PR DESCRIPTION
**Pull request by gbaraldi**
I'm not sure if this will work, but I thought the execution time was a bit odd.
I'm really bad at swift 😄

### CODE REVIEW

**Comment by Moelf** [comment]
fix #37 ?

File: src/leibniz.swift [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    pi += x / (2.0 * Double(i) - 1.0)
 }
 
-pi *= 4
-print(pi)
+pi *= 4.0
+print(String(format: "%.16f", pi))
 
 // TODO:
 //  * Read rounds from file
```
Comment (giordano) [inline src/leibniz.swift:16]: Can remove this TODO

**Review by niklas-heer — APPROVED** [review]

**Comment by gbaraldi** [comment]
That's odd, it does run locally for me, on both macos and linux.

**Comment by giordano** [comment]
What if you keep `print(pi)` as it was before?

**Comment by niklas-heer** [comment]
> That's odd, it does run locally for me, on both macos and linux.

Also, if you run it with `earthly --config earthly-config.yml +swift` or do you just run the program?

**Comment by gbaraldi** [comment]
https://stackoverflow.com/questions/57095407/swift-print-doesnt-appear-in-stdout-but-3rd-party-c-library-logs-do-when-runn this is what I'm suspicious of. 
I ran it at my terminal.

**Comment by niklas-heer** [comment]
@gbaraldi I tried it out in Codespaces, and it seems to still fail.
image

**Comment by gbaraldi** [comment]
This runs locally with earthly for me. My rounds.txt didn't have a new line :laughing:

**Comment by niklas-heer** [comment]
Yes, now it seems to work. I'll let the CI run in full then.
image

**Review by niklas-heer — APPROVED** [review]

**Comment by niklas-heer** [comment]
Thank you very much for your contribution @gbaraldi :+1:

**Pull request merged**